compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -84,72 +128,39 @@ internal class ComposeLayer(
         }
     }
 
-    private val platform = object : Platform by Platform.Empty {
-        override fun setPointerIcon(pointerIcon: PointerIcon) {
-            _component.cursor = (pointerIcon as? AwtCursor)?.cursor ?: Cursor(Cursor.DEFAULT_CURSOR)
-        }
-
-        override fun accessibilityController(owner: SemanticsOwner) =
-            AccessibilityControllerImpl(owner, _component, onFocusReceived = {
-                _component.requestNativeFocusOnAccessible(it)
-            })
+    protected val platform: DesktopPlatform = DesktopPlatform()
 
-        override val windowInfo = WindowInfoImpl()
+    internal val scene = ComposeScene(
+        MainUIDispatcher + coroutineExceptionHandler,
+        platform,
+        Density(1f),
```

Comment:
Why do we need it here? Usually I am against having "platform = platform", "density = Density"

Reply:
The point here was that labels are only for part of the parameters. Checked with coding convention, since it's not about repeating primitives - that's totally OK